docs/Troubleshooting/Troubleshooting.md

```diff
@@ -5,10 +5,49 @@ description: Frequently asked questions FAQ and answers for troubleshooting Pant
 
 If Pantheon is not working as expected, here are some things to check or try. 
 
-## Command Line Options
+## Supplied Genesis Block Does Not Match Stored Chain Data 
 
-Characters such as smart quotes and long (em) hyphens won't work in Pantheon command line options. Ensure quotes have
-not been automatically converted to smart quotes or double hyphens combined into em hyphens.
+If you get a `Supplied genesis block does not match stored chain data` error, delete the `database` directory in 
```

Comment:
Do you want to say "delete existing database", or use the genesis file which matches the genesis block of data directory? (in case someone blows away important data, we don't want to be liable).

Reply:
Updated